README.md

```diff
@@ -68,3 +74,45 @@ For instance, if you want your set of customized keybindings, you can just edit
     nmap <Leader>nx :call notes#toggle_checkbox(line('.'))<cr>
     " Fast Export function
     nmap <leader>ne :call notes#export() <cr>
+
+
+______
+
+
+Customizing templates
+---
+The plugin supports writing notes based on user defined templates.
+You can either define a particular path where a note template is stored, and set it in your vimrc:
+
+    ```
+    g:notes_template_path = <template_dir>/<template_file>
+    ```
+
+or let the plugin configure a default path and generate a sample template for you. In this case,
+you just need to set in your vimrc the following variable:
+
+    ```
+    g:notes_template_autogen = 1
+    ```
+
+and both the template directory and the sample file are generated according to the defaults
+specified in the table above.
+If you would like to keep this feature disabled, you can rely on the `g:notes_template_autogen` which
```

Comment:
maybe I can just say that given notes_template_autogen is 0, this feature has no effect